src/main.rs

```diff
@@ -241,6 +241,7 @@ async fn entry(jobserver_client: LazyJobserverClient) -> Result<()> {
     let desired_targets = get_desired_targets(&opts.targets);
 
     // Compute install directory
+    let custom_install_path = opts.install_path.is_some();
```

Comment:
I think we should do a further check that the install path if Some is not actually equal to `cargo_home().join("bin")`, as that would still be a global install. Even if a completely redundant option to pass, I could see it being used in scripts.

Reply:
> `cargo_home().join("bin")`

`home::cargo_home` can be overridden using environment variable `CARGO_HOME`.
Shall we consider this case to also be a global install?

Reply:
Done!